-------------Error at 07/23/2020 11:08:40------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 07/23/2020 11:08:40------------- 

-------------Fatal at 07/23/2020 11:08:41------------- 
 Stump.Server.BaseServer.ServerBase.HandleCrashException -> 
	 Crash Exception : Authentication to host 'mysql-quizeo.alwaysdata.net' for user 'quizeo' using method 'mysql_native_password' failed with message: Access denied for user 'quizeo'@'93.20.184.140' (using password: YES)
 Source: MySql.Data -> Void AuthenticationFailed(System.Exception)
 Stack Trace:
   à MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.AuthenticationFailed(Exception ex)
   à MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.ReadPacket()
   à MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.Authenticate(Boolean reset)
   à MySql.Data.MySqlClient.NativeDriver.Authenticate(String authMethod, Boolean reset)
   à MySql.Data.MySqlClient.NativeDriver.Open()
   à MySql.Data.MySqlClient.Driver.Open()
   à MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   à MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   à MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   à MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
   à MySql.Data.MySqlClient.MySqlPool.GetConnection()
   à MySql.Data.MySqlClient.MySqlConnection.Open()
   à Stump.ORM.Database.OpenSharedConnection() dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\Core\Stump.ORM\PetaPoco.cs:ligne 243
   à Stump.ORM.DatabaseAccessor.OpenConnection(Database database) dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\Core\Stump.ORM\DatabaseAccessor.cs:ligne 106
   à Stump.ORM.DatabaseAccessor.OpenConnection() dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\Core\Stump.ORM\DatabaseAccessor.cs:ligne 140
   à Stump.Server.AuthServer.AuthServer.Initialize() dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\server\Stump.Server.AuthServer\AuthServer.cs:ligne 109 
-------------Fatal at 07/23/2020 11:08:41------------- 

-------------Fatal at 07/23/2020 11:08:41------------- 
 Stump.Server.BaseServer.ServerBase.HandleCrashException -> 
	 Crash Exception : Access denied for user 'quizeo'@'93.20.184.140' (using password: YES)
 Source: MySql.Data -> MySql.Data.MySqlClient.MySqlPacket ReadPacket()
 Stack Trace:
   à MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   à MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.ReadPacket() 
-------------Fatal at 07/23/2020 11:08:41------------- 

-------------Error at 07/23/2020 11:08:58------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 07/23/2020 11:08:58------------- 

-------------Fatal at 07/23/2020 11:08:59------------- 
 Stump.Server.BaseServer.ServerBase.HandleCrashException -> 
	 Crash Exception : Authentication to host 'mysql-quizeo.alwaysdata.net' for user 'quizeo' using method 'mysql_native_password' failed with message: Access denied for user 'quizeo'@'93.20.184.140' (using password: YES)
 Source: MySql.Data -> Void AuthenticationFailed(System.Exception)
 Stack Trace:
   à MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.AuthenticationFailed(Exception ex)
   à MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.ReadPacket()
   à MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.Authenticate(Boolean reset)
   à MySql.Data.MySqlClient.NativeDriver.Authenticate(String authMethod, Boolean reset)
   à MySql.Data.MySqlClient.NativeDriver.Open()
   à MySql.Data.MySqlClient.Driver.Open()
   à MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   à MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   à MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   à MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
   à MySql.Data.MySqlClient.MySqlPool.GetConnection()
   à MySql.Data.MySqlClient.MySqlConnection.Open()
   à Stump.ORM.Database.OpenSharedConnection() dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\Core\Stump.ORM\PetaPoco.cs:ligne 243
   à Stump.ORM.DatabaseAccessor.OpenConnection(Database database) dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\Core\Stump.ORM\DatabaseAccessor.cs:ligne 106
   à Stump.ORM.DatabaseAccessor.OpenConnection() dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\Core\Stump.ORM\DatabaseAccessor.cs:ligne 140
   à Stump.Server.AuthServer.AuthServer.Initialize() dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\server\Stump.Server.AuthServer\AuthServer.cs:ligne 109 
-------------Fatal at 07/23/2020 11:08:59------------- 

-------------Fatal at 07/23/2020 11:08:59------------- 
 Stump.Server.BaseServer.ServerBase.HandleCrashException -> 
	 Crash Exception : Access denied for user 'quizeo'@'93.20.184.140' (using password: YES)
 Source: MySql.Data -> MySql.Data.MySqlClient.MySqlPacket ReadPacket()
 Stack Trace:
   à MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   à MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.ReadPacket() 
-------------Fatal at 07/23/2020 11:08:59------------- 

-------------Error at 07/23/2020 11:10:12------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 07/23/2020 11:10:12------------- 

-------------Error at 07/23/2020 11:18:07------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 07/23/2020 11:18:07------------- 

-------------Error at 07/23/2020 20:39:55------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 07/23/2020 20:39:55------------- 

-------------Error at 07/23/2020 21:03:41------------- 
 Stump.Server.BaseServer.ServerBase.KeepSQLConnectionAlive -> 
	Cannot ping SQL connection : {0} 
-------------Error at 07/23/2020 21:03:41------------- 

-------------Warn at 07/23/2020 21:03:41------------- 
 Stump.Server.BaseServer.ServerBase.KeepSQLConnectionAlive -> 
	Try to Re-open the connection 
-------------Warn at 07/23/2020 21:03:41------------- 

-------------Error at 07/23/2020 21:04:03------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : System.InvalidOperationException: Connection must be valid and open.
   à MySql.Data.MySqlClient.Interceptors.ExceptionInterceptor.Throw(Exception exception)
   à MySql.Data.MySqlClient.MySqlConnection.Throw(Exception ex)
   à MySql.Data.MySqlClient.MySqlCommand.Throw(Exception ex)
   à MySql.Data.MySqlClient.MySqlCommand.CheckState()
   à MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   à MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   à System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   à Stump.ORM.Database.<Query>d__71`1.MoveNext() dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\Core\Stump.ORM\PetaPoco.cs:ligne 783
   à System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   à System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   à Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\server\Stump.Server.AuthServer\Managers\AccountManager.cs:ligne 96
   à Stump.Core.Timers.TimedTimerEntry.Trigger() dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\Core\Stump.Core\Timers\TimedTimerEntry.cs:ligne 128
   à Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:ligne 223. 
-------------Error at 07/23/2020 21:04:03------------- 

-------------Error at 07/23/2020 21:04:04------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : System.InvalidOperationException: Connection must be valid and open.
   à MySql.Data.MySqlClient.Interceptors.ExceptionInterceptor.Throw(Exception exception)
   à MySql.Data.MySqlClient.MySqlConnection.Throw(Exception ex)
   à MySql.Data.MySqlClient.MySqlCommand.Throw(Exception ex)
   à MySql.Data.MySqlClient.MySqlCommand.CheckState()
   à MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   à MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   à System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   à Stump.ORM.Database.<Query>d__71`1.MoveNext() dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\Core\Stump.ORM\PetaPoco.cs:ligne 783
   à System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   à System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   à Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\server\Stump.Server.AuthServer\Managers\AccountManager.cs:ligne 96
   à Stump.Core.Timers.TimedTimerEntry.Trigger() dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\Core\Stump.Core\Timers\TimedTimerEntry.cs:ligne 128
   à Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:ligne 223. 
-------------Error at 07/23/2020 21:04:04------------- 

-------------Error at 07/23/2020 21:18:58------------- 
 Stump.Server.AuthServer.IPC.IPCClient.SendError -> 
	IPC error : {0} 
-------------Error at 07/23/2020 21:18:58------------- 

-------------Error at 07/23/2020 21:20:02------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 07/23/2020 21:20:02------------- 

-------------Error at 07/23/2020 21:54:16------------- 
 Stump.Server.BaseServer.ServerBase.KeepSQLConnectionAlive -> 
	Cannot ping SQL connection : {0} 
-------------Error at 07/23/2020 21:54:16------------- 

-------------Warn at 07/23/2020 21:54:16------------- 
 Stump.Server.BaseServer.ServerBase.KeepSQLConnectionAlive -> 
	Try to Re-open the connection 
-------------Warn at 07/23/2020 21:54:16------------- 

-------------Error at 07/23/2020 21:54:27------------- 
 Stump.Server.BaseServer.ServerBase.KeepSQLConnectionAlive -> 
	Cannot reopen the SQL connection : {0} 
-------------Error at 07/23/2020 21:54:27------------- 

-------------Error at 07/23/2020 21:54:27------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : System.InvalidOperationException: Connection must be valid and open.
   à MySql.Data.MySqlClient.Interceptors.ExceptionInterceptor.Throw(Exception exception)
   à MySql.Data.MySqlClient.MySqlConnection.Throw(Exception ex)
   à MySql.Data.MySqlClient.MySqlCommand.Throw(Exception ex)
   à MySql.Data.MySqlClient.MySqlCommand.CheckState()
   à MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   à MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   à System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   à Stump.ORM.Database.<Query>d__71`1.MoveNext() dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\Core\Stump.ORM\PetaPoco.cs:ligne 783
   à System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   à System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   à Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\server\Stump.Server.AuthServer\Managers\AccountManager.cs:ligne 96
   à Stump.Core.Timers.TimedTimerEntry.Trigger() dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\Core\Stump.Core\Timers\TimedTimerEntry.cs:ligne 128
   à Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:ligne 223. 
-------------Error at 07/23/2020 21:54:27------------- 

-------------Error at 07/23/2020 21:54:27------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : System.InvalidOperationException: Connection must be valid and open.
   à MySql.Data.MySqlClient.Interceptors.ExceptionInterceptor.Throw(Exception exception)
   à MySql.Data.MySqlClient.MySqlConnection.Throw(Exception ex)
   à MySql.Data.MySqlClient.MySqlCommand.Throw(Exception ex)
   à MySql.Data.MySqlClient.MySqlCommand.CheckState()
   à MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   à MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   à System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   à Stump.ORM.Database.<Query>d__71`1.MoveNext() dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\Core\Stump.ORM\PetaPoco.cs:ligne 783
   à System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   à System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   à Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\server\Stump.Server.AuthServer\Managers\AccountManager.cs:ligne 96
   à Stump.Core.Timers.TimedTimerEntry.Trigger() dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\Core\Stump.Core\Timers\TimedTimerEntry.cs:ligne 128
   à Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) dans C:\Users\vince\OneDrive\Bureau\Code\dofus\Populus 2.51\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:ligne 223. 
-------------Error at 07/23/2020 21:54:27------------- 

-------------Error at 07/23/2020 22:05:15------------- 
 Stump.Server.AuthServer.IPC.IPCClient.SendError -> 
	IPC error : {0} 
-------------Error at 07/23/2020 22:05:15------------- 

-------------Error at 07/23/2020 22:07:55------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 07/23/2020 22:07:55------------- 

